### PR TITLE
feat(test): Run storyshots on specific components

### PR DESCRIPTION
**Pull request by Thebarda**
Here are scripts examples : 
- `npm run test:storyshot "Breadcrumb Title"` : run storyshots tests for Breadcrumb and Title
- `npm run test:storyshot` : run all storyshots tests

### CODE REVIEW

File: src/storyshots.test.js [hunk 1]
```diff
@@ -15,8 +15,18 @@ const beforeScreenshot = () => {
   );
 };
 
+const idxStoryArg = process.argv.indexOf('-story');
+let regexp = null;
+if (idxStoryArg !== -1) {
+  const storyComponents = process.argv.slice(idxStoryArg + 1);
+  const componentsToTest =
+    storyComponents.length > 0 ? storyComponents[0].replace(/ /g, '|') : '';
```
Comment (bdauria) [inline src/storyshots.test.js]: Why do we need this replacement?

Comment (Thebarda) [reply src/storyshots.test.js]: Here is the syntax to test multiple components : `npm run test:storyshot "Breadcrumb Title"`
So I have to create a regexp that tells storyshot to test these components and I have to make a 'OR' by replacing spaces by '|'. 
But I can remove this replacement thus the new syntax will be : `npm run test:storyshot "Breadcrumb|Title"`

Comment (bdauria) [reply src/storyshots.test.js]: I see. the '|' character works fine for me.

Comment (Thebarda) [reply src/storyshots.test.js]: Yeah, it's just command style. Thus, if you prefer '|' in command, go for it !

File: src/storyshots.test.js [hunk 2]
```diff
@@ -15,8 +15,17 @@ const beforeScreenshot = () => {
   );
 };
 
+const idxStoryArg = process.argv.indexOf('-story');
```
Comment (bdauria) [inline src/storyshots.test.js]: Another approach:

```
const getStoryKindRegex = () => {
  const singleStoryArgumentIndex = process.argv.indexOf('-story');
  
  if (singleStoryArgumentIndex === -1) {
    return null;
  }

  const givenStories = process.slice(singleStoryArgumentIndex + 1);

  if (givenStories.length === 0) {
    return null;
  }

  const [componentsToTest] =  givenStories:
  
  return new RegExp(`^(${componentsToTest || ''})$`, 'g');
}
```

File: package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
     "start:dev": "npm run bundle -- --watch --source-maps",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "build:storybook": "build-storybook -c .storybook -o .out",
-    "test": "jest"
+    "test": "jest",
+    "test:storyshot": "npm test storyshots.test.js -- -story"
```
Comment (kduret) [inline package.json]:
```suggestion
    "test:storyshot": "npm test storyshots.test.js -- --story"
```

File: src/storyshots.test.js [hunk 4]
```diff
@@ -15,8 +15,22 @@ const beforeScreenshot = () => {
   );
 };
 
+const getStoryKindRegex = () => {
+  const singleStoryArgumentIndex = process.argv.indexOf('-story');
```
Comment (kduret) [inline src/storyshots.test.js]:
```suggestion
  const singleStoryArgumentIndex = process.argv.indexOf('--story');
```

File: package.json (hunk 3 above)
Comment (Thebarda) [reply package.json]: A simple question : Why do we need another dash ?

Comment (kduret) [reply package.json]: https://medium.com/@nickcis/jest-passing-custom-arguments-d44ef3f2defb

File: README.md [hunk 5]
```diff
@@ -1,3 +1,50 @@
...
+
+`npm run eslint`
+
+You can also fix fixable linter errors by running:
+
+`npm run eslint:fix`
+
+# Storybook
+
+You are using Storybook to visualize our components through stories.
```
Comment (bdauria) [inline README.md]:
```suggestion
We are using Storybook to visualize our Components through stories.
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,50 @@
...
+
+or
+
+`npm t`
+
+To run End to End tests:
+  - Build Storybook : `npm run build:storybook`
+  - Run all Storyshot tests : `npm run test:storyshot`
+
+You can also test one or more component using the following syntax:
```
Comment (bdauria) [inline README.md]:
```suggestion
You can also test one or more Components using the following syntax:
```

**Pull request merged**